fxa-client/src/lib.rs

```diff
@@ -498,24 +503,91 @@ impl FirefoxAccount {
         self.state.config.token_server_endpoint_url()
     }
 
-    pub fn handle_push_message(&self) {
-        panic!("Not implemented yet!")
+    /// Retrieve our own client instance metadata.
+    pub fn client_instance(&mut self) -> Result<ClientInstanceResponse> {
+        let refresh_token = self.get_refresh_token()?;
+        let client = Client::new(&self.state.config);
+        client.client_instance(&refresh_token)
+    }
+
+    /// Retrieve the metadata from all the clients instances connected to
+    /// the account (including ours).
+    pub fn clients_instances(&mut self) -> Result<Vec<ClientInstanceResponse>> {
```

Comment:
What naming convention do we use for the public-facing API here?  Is it typically "verb-noun" or similar?  I'm wondering if these should be `get_client_instances`, `get_pending_commands` etc.